scenario_player/tasks/base.py

```diff
@@ -110,9 +110,12 @@ def __call__(self, *args, **kwargs):
         self._runner.running_task_count += 1
         self._start_time = time.monotonic()
         try:
+            timeout_s = None
+            # Config can be something else than a dictionary
```

Comment:
when would the config be something else? this seems a problem that should be handled in the caller site, and not here.

Reply:
For example in the `WaitTask`, the config is just an `int`

Reply:
The point is that it should be fixed there then.

Reply:
I guess we want to support this:
```
 - wait: 30
```
instead of 
```
- wait: {time: 30}
```

Reply:
I propopose to fix that later: https://github.com/raiden-network/scenario-player/issues/584